masterlist.yaml

```
@@ -1417,6 +1417,12 @@ plugins:
       - crc: 0xE86A8F90
         util: 'FO4Edit v4.0.3j (RC2 for 4.0.4)'
 
+  - name: 'Realistic Reverb and Ambiance.esp'
+    url:
+      - link: 'https://www.nexusmods.com/fallout4/mods/61140/'
+        name: 'Realistic Reverb and Ambience Overhaul - VR and FP'
+    after: [ 'Ambient Wasteland.esp' ]
```

Since this plugin is required, `req` would be more appropriate. LOOT will sort plugins after their requirements as well, just like masters:

```suggestion
    req: [ 'Ambient Wasteland.esp' ]
```